internal/image/tag_selector_ghcr_test.go

```diff
@@ -18,7 +18,8 @@ import (
 // may get rate-limited executing these tests, so they're disabled by default.
 
 func TestSelectTagGHCR(t *testing.T) {
-	const kargoRepo = "ghcr.io/akuity/kargo"
+	const guestbookRepo = "ghcr.io/jessesuen/guestbook"
+	const guestbookTag = "v0.0.47"
```

Comment:
Hmm, I'd like the freedom to delete this repo without breaking tests. I've repushed this multi-arch image as: `ghcr.io/akuity/kargo-test:unknown`. Can we change this to use that?

https://github.com/orgs/akuity/packages/container/kargo-test/156938778?tag=unknown